### PR TITLE
Make It Clear Issue Has Been Crowdfunded V2

### LINKED ISSUE #2164: Make It Clear Issue Has Been Crowdfunded V2
### User Story

As a user I'd like to easily see that an issue has been crowdfunded.

### Defintion of Done

- [ ] If an issue has been crowdfunded iwth addtional tokens, display the tokens icons and a + next to the initial value.
- [ ] Add See More Link and Carat to display additional tokens. This should collapse and expand.
- [ ]  Clicking on the entire row should still go to the issue details page.
- [ ] On the row, show ETH first

- [ ] The USD value should show **total USD conversions for the initial funding and tokens** that do have a USD conversion.
- [ ] On the issue details, move the funded amount under the Issue title, and display all tokens in a line, with USD last.
- [ ] Shrink the font and height of the token value tag to 24px

### Review with @Pixelandesign 

crowdfunded3

crowdfunded2

### PR DESCRIPTION
**Pull request by pinkiebell**
Fixes #2164

### CODE REVIEW

**Comment by pinkiebell** [comment]
screenshot 2018-09-07 at 02 09 57
screenshot 2018-09-07 at 08 23 49

File: app/assets/v2/js/shared.js [hunk 1]
```diff
@@ -955,63 +955,107 @@ var usdToAmount = function(event) {
   });
 };
 
-function renderBountyRowsFromResults(results) {
+function renderBountyRowsFromResults(results, renderForExplorer) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/shared.js]: ❤️

File: app/assets/v2/images/coin.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
+        <g id="Artboard-Copy-3" transform="translate(-1223.000000, -560.000000)" fill="#FFFFFF" stroke="#FF7200" stroke-width="0.7">
+            <g id="Group" transform="translate(1224.000000, 561.000000)">
+                <path d="M6,5 C6,2.23869565 8.23869565,0 11,0 C13.7613043,0 16,2.23869565 16,5 C16,7.76130435 13.7613043,10 11,10 C8.23869565,10 6,7.76130435 6,5 Z" id="Stroke-3317-Copy-21"></path>
+                <path d="M3,5 C3,2.23869565 5.23869565,0 8,0 C10.7613043,0 13,2.23869565 13,5 C13,7.76130435 10.7613043,10 8,10 C5.23869565,10 3,7.76130435 3,5 Z" id="Stroke-3317-Copy-22"></path>
+                <path d="M0,5 C0,2.23869565 2.23869565,0 5,0 C7.76130435,0 10,2.23869565 10,5 C10,7.76130435 7.76130435,10 5,10 C2.23869565,10 0,7.76130435 0,5 Z" id="Stroke-3317-Copy-23"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (thelostone-mc) [inline app/assets/v2/images/coin.svg]: @pinkiebell can we run `svgo` against this file ?

**Comment by pinkiebell** [comment]
@thelostone-mc svgo-ing Done 👍

**Comment by pinkiebell** [comment]
Side note: I labelled this pr as wip because of one open question:
  - How do we handle tokens with no usd conversion?
     Right now we have no indicator for that and fixing requires backend changes,
     do we want to handle that case in this PR or defer?

**Comment by PixelantDesign** [comment]
@pinkiebell could we display the token with no usd conversion in the tool tip but not average in the final USD value?

**Comment by pinkiebell** [comment]
Updates
screenshot 2018-09-08 at 00 54 17
screenshot 2018-09-08 at 00 54 28

**Comment by pinkiebell** [comment]
The only thing what's left is the caret(arrow) with the right color(same as link color).
I'm also open for another round of improvements 👍

**Comment by thelostone-mc** [comment]
@pinkiebell could you rebase it once ? we'll put this up on staging and test it out there today

**Comment by pinkiebell** [comment]
@thelostone-mc Done.

Errors still in this PR are likely:
The tooltip misses the 'as of'  <time>.
I'm a little bit confused with the '$' sign in the crowdfunding tooltip,
because I always think of dollar valuation if I see the '$', was that actually right or is this wrong to convert the crowdfunding token value to dollar in the tooltip? 🤔

**Comment by pinkiebell** [comment]
@thelostone-mc 
Any news?

**Comment by SaptakS** [comment]
@pinkiebell I will be reviewing it in some time and get back to you. :)

**Comment by SaptakS** [comment]
@pinkiebell I am seeing this in every non-crowfunded bounty details page. Can you please check?
screen shot 2018-09-24 at 4 04 16 pm

**Comment by pinkiebell** [comment]
@SaptakS
I can't reproduce this behaviour, what wonders me is the 'as ofnow' in the USD tag.
That looks like the JS code is not from this PR. Maybe a cache issue?

@mbeacom 
I addressed your comments. I did a rebase though, hope that's ok.

**Comment by pinkiebell** [comment]
@SaptakS
Fun times 😄 . This may be problem with docker and inotify events.
Just reproduced that in the issue explorer because I switched branches/rebased and the django app didn't reload (fs inotify events).

**Comment by SaptakS** [comment]
@pinkiebell The icon beside `See All` is opposite according to me. It should point downwards when it is closed and upwards when it is open. cc: @thelostone-mc @mbeacom @PixelantDesign

File: app/assets/v2/js/pages/bounty_details.js [hunk 3]
```diff
@@ -225,60 +221,67 @@ var callbacks = {
...
-          crowdfunded_tokens.push({
-            amount: funding,
-            token: token
-          });
-        }
-        const template = '<p class="m-0" style="color:#fb9470">+' + funding + ' ' + token + ' in crowdfunding</p>';
 
-        tooltip_info.push(template);
-      }
+    const decimals = 2;
```
Comment (SaptakS) [inline app/assets/v2/js/pages/bounty_details.js]: We are showing 3 decimal places in almost all places. Any reason why we are showing only 2 for crowdfunded amount?

Comment (pinkiebell) [reply app/assets/v2/js/pages/bounty_details.js]: @SaptakS Aesthetics? 😄 Fixed!

**Comment by pinkiebell** [comment]
@SaptakS Fixed the arrow rotation. But don't ask me why I wrote accordion in the commit message 🤣 .

**Comment by PixelantDesign** [comment]
@SaptakS could you put this on staging please?

**Comment by SaptakS** [comment]
@PixelantDesign it's on staging at https://stage.gitcoin.co/

**Comment by PixelantDesign** [comment]
@pinkiebell could we left align the USD values?

screen shot 2018-09-24 at 10 28 14 pm

**Comment by pinkiebell** [comment]
@PixelantDesign 
1. That's the ETH amount in USD, we should probably change the formatting?
2. Did you mean putting the green USD tag on the left or should they all be aligned to the right for example? That could be a little bit tricky with our current CSS if we want to make sure that they are all aligned top-down.

@mbeacom 
Hehe, we can tell future prices, nice! I will put a fix in 👍

**Comment by pinkiebell** [comment]
one
two

**Comment by pinkiebell** [comment]
@mbeacon @SaptakS @PixelantDesign 
FInal review? 🙏

**Comment by thelostone-mc** [comment]
@pinkiebell one final rebase and then we can throw it on staging

**Comment by pinkiebell** [comment]
@thelostone-mc 
Done 👍

**Comment by PixelantDesign** [comment]
Looking good to me...One thing I see... 

It should not be possible to have 791 cents.

screen shot 2018-10-01 at 11 11 01 am

**Comment by pinkiebell** [comment]
@PixelantDesign 
Resolved 👍 . We now display token amounts with max .3 decimals and dollar conversion .2 decimals.

**Pull request merged**